root/etc/e-smith/events/actions/nethserver-mattermost-conf

```diff
@@ -20,21 +20,54 @@
 # along with NethServer.  If not, see COPYING.
 #
 
-su - postgres -c "scl enable rh-postgresql94 -- psql --port=55432 -lqt | cut -d \| -f 1 | grep -q -w mattermost"
+su - postgres -c "scl enable rh-postgresql12 -- psql --port=55434 -lqt | cut -d \| -f 1 | grep -q -w mattermost"
 if [ $? -eq 1 ]; then # database do not exists
+    su - postgres -c "scl enable rh-postgresql94 -- psql --port=55432 -lqt | cut -d \| -f 1 | grep -q -w mattermost"
+    if [ $? -eq 0 ]; then
+        echo "Migrating data from postgresql 9.4 to postgresql 12..."
+        systemctl stop rh-postgresql94-postgresql
+        systemctl stop rh-postgresql12-postgresql
+        su - postgres -c "export LD_LIBRARY_PATH=/opt/rh/rh-postgresql94/root/usr/lib64:/opt/rh/rh-postgresql12/root/usr/lib64:$LD_LIBRARY_PATH && scl enable rh-postgresql12 -- pg_upgrade -b /opt/rh/rh-postgresql94/root/usr/bin -B /opt/rh/rh-postgresql12/root/usr/bin -d /var/opt/rh/rh-postgresql94/lib/pgsql/data -D /var/opt/rh/rh-postgresql12/lib/pgsql/data"
+        if [ $? -eq 0 ]; then
+            echo "Migration completed successfully"
```

Comment:
This messages is not usefull: if everything goes well, the action should simply exit with 0 code.

Reply:
I would report what happened anyway because it is a one-shot action, so we can distinguish what the action did.

```suggestion
            echo "[NOTICE] Migration completed successfully"
```